CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# How to Contribute to Jaeger
+
+We'd love your help!
+
+Jaeger is [Apache 2.0 licensed](LICENSE) and accepts contributions via GitHub pull requests. This document outlines some of the conventions on development workflow, commit message formatting, contact points and other resources to make it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+## Developing
+
+1. `npm install`
```

Comment:
Which Node version?

Reply:
I wrote it this way as jaeger-client-node had similar instructions